src/animationIcons/CheckIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import type { SVGProps } from 'react';
+import { styled } from '../../stitches.config';
+
+const Path = styled('path', {
+  'input:checked ~ div &': {
+    transition: 'stroke-dashoffset 1s cubic-bezier(0.16, 1, 0.3, 1)',
+  },
+  transition: 'stroke-dashoffset 0.2s cubic-bezier(0.11, 0, 0.5, 0)',
+});
```

Comment:
I think we should use [variants](https://stitches.dev/docs/variants) to essentially add a `isChecked` prop that would adjust the transition style.

The reason I would do this is that the style then relies on a prop and not the structure of the DOM.

Right now this styling is very conditional based on where it is used and isn't really sharable.

Thoughts @Boeing787 and @lukerohanbailey?

Reply:
Agreed! See comment https://github.com/contra/ui-kit/pull/53#issuecomment-1012642484